database/migrations/2019_05_03_000003_create_receipts_table.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateReceiptsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('receipts', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('billable_id');
+            $table->string('billable_type');
+            $table->unsignedBigInteger('paddle_subscription_id')->nullable()->index();
+            $table->string('checkout_id');
+            $table->string('order_id')->unique();
+            $table->string('amount');
+            $table->string('tax');
+            $table->string('currency', 3);
+            $table->integer('quantity');
+            $table->string('download_url')->unique();
```

Comment:
This is always present even if there is no download?

Reply:
Yes there's always a receipt.

Reply:
So, what is the value of the `download_url`?

Reply:
This is the link to the receipt pdf. Every payment has its own receipt pdf. It can never be null. Maybe I should rename it?

Reply:
Hmm, it looks like the Paddle payload is "receipt_url"... maybe we should just call it that?

Reply:
Definitely. I actually named it like that at first but thought download would be more clear what it was. But on second thought I agree that `receipt_url` makes more sense. It's renamed now 👍